interleave.el

```diff
@@ -233,12 +238,17 @@ SPLIT-WINDOW is a function that actually splits the window, so it must be either
   (let* ((buf (current-buffer))
          (pdf-file-name
           (or (interleave--headline-pdf-path buf)
-              (interleave--find-pdf-path buf)
-              (let ((filename
-                     (read-file-name "No #+INTERLEAVE_PDF property found. Please specify path: " "~/")))
-                (with-current-buffer buf
-                  (insert "#+INTERLEAVE_PDF: " filename))
-                filename))))
+              (interleave--find-pdf-path buf))))
+    (when (not pdf-file-name)
```

Comment:
Minor nitpick: Can you make this an `unless`?

Reply:
Oh yes, of course, I completely forget it exists!